README.md

```diff
@@ -273,11 +273,13 @@ spec:
 
 #### ServiceBindingRequestStatus
 
-`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status
+`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

Comment:
Doesn't the ServiceBindingRequestStatus depict the status of the Service Binding Request and not the Service Binding operator? The operator can support multiple ServiceBindingRequests. It seems unusual that the status of each ServiceBindingRequest CR would somehow contribute to the overall status of the operator.

Reply:
Yes the `status` depicts the status of the specific request. A failure might indicate that the binding actors were missing, for example.